tests/integrations/config/mod.rs

```diff
@@ -16,7 +16,7 @@ use engine_traits::PerfLevel;
 use file_system::{IOPriority, IORateLimitMode};
 use kvproto::encryptionpb::EncryptionMethod;
 use pd_client::Config as PdConfig;
-use raft_log_engine::RecoveryMode;
+use raft_log_engine::{ReadableSize as RaftEngineReadableSize, RecoveryMode};
```

Comment:
why is `ReadableSize` in raft_log_engine

Reply:
Config is defined in Raft Engine, tikv's ReadableSize is copied there.